### PR TITLE
add sepolicy for evs

### PR DESCRIPTION
**Pull request by shivasku82**

### CODE REVIEW

**Review by dyang23 — APPROVED** [review]

**Review by balajim001 — APPROVED** [review]
LGTM

**Review by dyang23 — APPROVED** [review]

**Review by neofang7 — APPROVED** [review]
LGTM

**Review by balajim001 — APPROVED** [review]
LGTM

File: car/hal_vehicle_default.te [hunk 1]
```diff
@@ -7,6 +7,7 @@ binder_call(hal_audio_default, hal_vehicle_default)
 binder_call(hal_vehicle_default, hal_audio_default)
 hal_client_domain(hal_vehicle_default, hal_audio)
 
+vendor_internal_prop(vendor_evs_hal_mesa_prop)
```
Comment (ceiba1985) [inline car/hal_vehicle_default.te]: would better put this property.te

File: car/hal_vehicle_default.te [hunk 2]
```diff
@@ -23,3 +24,15 @@ ignore_adb_debug(`hal_vehicle_default')
 
 allow hal_vehicle_default serial_device:chr_file rw_file_perms;
 allow hal_vehicle_default sysfs_power:file rw_file_perms;
+allow hal_evs_default gpu_device:dir { read open};
+allow hal_evs_default sysfs_app_readable:file { read open getattr};
+allow hal_evs_default sysfs_gpu:dir search;
+allow hal_evs_default graphics_config_writable_prop:file { read open getattr map};
```
Comment (ceiba1985) [inline car/hal_vehicle_default.te]: get_prop(hal_evs_default, graphics_config_writable_prop)

File: car/hal_vehicle_default.te [hunk 3]
```diff
@@ -23,3 +24,15 @@ ignore_adb_debug(`hal_vehicle_default')
...
+allow hal_evs_default traced_producer_socket:sock_file write;
+allow hal_evs_default traced:unix_stream_socket connectto;
```
Comment (ceiba1985) [inline car/hal_vehicle_default.te]: do we really need line 31 and 32? if don't we can replate allow with dontaudit

**Review by balajim001 — APPROVED** [review]
LGTM

**Review by swaroopbalan — APPROVED** [review]

**Review by ceiba1985 — APPROVED** [review]

**Pull request merged**